### ISSUE TITLE
pwdLastSet attribute no longer accepts timestamps in mocks

### ISSUE THREAD

**Issue by djlambert** [cannatag/ldap3#512]
In v2.4.1 the pwdLastSet attribute validator was changed to only allow a -1 value (and soon a 0 value #484)

Now when using the MOCK_SYNC strategy I get a validation error when loading my previously dumped json.

```
ldap3.core.exceptions.LDAPInvalidValueError: value '130124221197470890' non valid for attribute 'pwdLastSet'

**Comment by cannatag** [comment]
You're right. In mock strategies values don't need validation when loading from a json dump.
Now validation is not applied when using ``entries_from_json()``. 

Can you check the code in dev?

Thanks,
Giovanni

**Comment by djlambert** [comment]
Issue no longer present in dev. Thanks!

**Comment by cannatag** [comment]
Fixed in 2.5
